pages/common/qc.md

```diff
@@ -0,0 +1,24 @@
+# qc
+
+> CLI snippet manager to execute command snippets stored in notes in QOwnNotes.
```

Comment:
```suggestion
> CLI snippet manager to execute command snippets stored in notes in QOwnNotes.
> See also: `qownnotes`.
```
What do you think of suggesting the page I've added?

Reply:
Hah, sure! I just now realized qownnotes has an entry 😄 thank you